src/encoded/schemas/file.json

```diff
@@ -1618,6 +1618,7 @@
                         "fine-mapped variants",
                         "fold change over control",
                         "footprints",
+                        "functional conservation scores",
```

Comment:
can we change it to "functional conservation quantifications"?

Reply:
Changed